src/entries/popup/components/DappIcon/DappIcon.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, Inline, Symbol } from '~/design-system';
+import { BoxStyles } from '~/design-system/styles/core.css';
+
+import ExternalImage from '../ExternalImage/ExternalImage';
+
+type DappIconSize = '60px' | '36px' | '32px' | '18px' | '16px' | '14px';
+
+const SYMBOL_SIZE = {
+  '60px': 32,
+  '36px': 18,
+  '32px': 16,
+  '18px': 10,
+  '16px': 8,
+  '14px': 8,
+};
+
+const RADIUS_SIZE = {
+  '60px': '18px' as BoxStyles['borderRadius'],
+  '36px': '10px' as BoxStyles['borderRadius'],
+  '32px': '9px' as BoxStyles['borderRadius'],
+  '18px': '6px' as BoxStyles['borderRadius'],
+  '16px': '4px' as BoxStyles['borderRadius'],
+  '14px': '4px' as BoxStyles['borderRadius'],
+};
```

Comment:
I think `satisfies Record<string, BoxStyles['borderRadius']>` would work too, so we don't need to `as BoxStyles['borderRadius']` it all

Reply:
nice, i added 

```
Record<DappIconSize, BoxStyles['borderRadius']>
```